### PR TITLE
Add links to guides in various places

### LINKED ISSUE #4392: Add links to help pages in requests
* On the host side: when you receive a request, there should be a helpful little text near the accept/reject buttons directing the user to a help page on how to respond to a request. "Not sure how to respond? Have a look at our guide here." or similar
* On the surfer side: there should be some kind of help text when writing a request "Read our guide to writing a great request" or similar
* On the surfer side, if they get rejected, there should be a link saying "Need help having your requests accepted? Read our guide here for the best tips!"

### LINKED ISSUE #4393: Help page: how to/guide/tips on responding to a request
Part of #4392 

Things like
* You don't need to feel pressure to accept
* Examples of good, succinct rejections
* etc

@jesseallhands assigning you for now but maybe I/someone else can help

### PR DESCRIPTION
**Pull request by aapeliv**
This is ready for frontend review: @jesseallhands and I are still working to finish off the guides and tweak the exact wording!

Closes #4392, closes #4393.

WIP guides:
- [Quick reference: writing great requests](https://chatwoot.couchershq.org/app/accounts/2/portals/couchersorg-help-center/en/articles/57)
- [How to send a host a request that gets accepted (with examples)](https://chatwoot.couchershq.org/app/accounts/2/portals/couchersorg-help-center/en/articles/41)
- [How should I respond to a request?](https://chatwoot.couchershq.org/app/accounts/2/portals/couchersorg-help-center/en/articles/56)
- [How do I create a great profile?](https://chatwoot.couchershq.org/app/accounts/2/portals/couchersorg-help-center/en/articles/55)

**Web frontend checklist**
- [ ] Formatted my code with `yarn format`
- [ ] There are no warnings from `yarn lint --fix`
- [ ] There are no console warnings when running the app
- [ ] Added any new components to storybook
- [ ] Added tests where relevant
- [ ] All tests pass
- [ ] Clicked around my changes running locally and it works
- [ ] Checked Desktop, Mobile and Tablet screen sizes

Screenshot from 2024-09-09 18-34-31
Screenshot from 2024-09-09 18-31-01
Screenshot from 2024-09-09 18-47-28
Screenshot from 2024-09-09 19-00-11
Screenshot from 2024-09-10 00-56-51

### CODE REVIEW

File: app/web/features/messages/requests/HostRequestSendField.tsx [hunk 1]
```diff
@@ -131,6 +133,42 @@ export default function HostRequestSendField({
 
   return (
     <form onSubmit={onSubmit}>
+      {isHost &&
+        hostRequest.status ===
+          HostRequestStatus.HOST_REQUEST_STATUS_PENDING && (
```
Comment (nabramow) [inline app/web/features/messages/requests/HostRequestSendField.tsx]: What about making this a a variable `isPendingHost` and reference that instead for readability?

File: app/web/features/messages/requests/HostRequestSendField.tsx [hunk 2]
```diff
@@ -131,6 +133,42 @@ export default function HostRequestSendField({
...
+                  how to respond to a request
+                </StyledLink>
+                .
+              </Trans>
+            </Typography>
+          </div>
+        )}
+      {!isHost &&
+        hostRequest.status ===
+          HostRequestStatus.HOST_REQUEST_STATUS_REJECTED && (
```
Comment (nabramow) [inline app/web/features/messages/requests/HostRequestSendField.tsx]: Same here maybe `isRejectedHost`. Just a nit.

**Review by nabramow — DISMISSED** [review]
I didn't test all of them, but I checked the declined surf request and that worked well and codewise looks good so I will trust you there!

File: app/web/features/messages/requests/HostRequestSendField.tsx (hunk 1 above)
Comment (aapeliv) [reply app/web/features/messages/requests/HostRequestSendField.tsx]: Done!

**Comment by jesseallhands** [comment]
Since the final design will look like this, I am hesitant to add too much text:
image

So I will suggest a shortened text for these links.

**Comment by jesseallhands** [comment]
To limit length, the text should be shortened to:

> [Read our guide](https://help.couchers.org/hc/couchersorg-help-center/articles/1725943310-quick-reference-writing-great-requests) on how to write a request that will get accepted.

The link takes you to an (objective) quick guide which also provides a link to a more detailed (subjective) guide with examples.

The text should appear on the desktop request page like this:

image

Mobile design to follow.

**Comment by aapeliv** [comment]
@jesseallhands I have updated according to your preferences even though I prefer mine :)

File: app/web/features/profile/edit/EditProfile.tsx [hunk 3]
```diff
@@ -155,6 +156,20 @@ export default function EditProfileForm() {
       )}
       {user ? (
         <>
+          <div className={classes.helpTextContainer}>
+            <Typography variant="body1">
+              <Trans i18nKey="profile:edit_profile_helper_text">
+                Looking for some inspiration on where to start?{" "}
+                <StyledLink
+                  variant="body1"
+                  href="https://help.couchers.org/hc/couchersorg-help-center/articles/1715658357-how-to-write-a-request-that-gets-accepted"
```
Comment (jesseallhands) [inline app/web/features/profile/edit/EditProfile.tsx]:
```suggestion
                  href="https://help.couchers.org/hc/couchersorg-help-center/articles/1725919197-how-do-i-create-a-great-profile"
```
This had the wrong hyperlink

**Review by jesseallhands — COMMENTED** [review]
Just one hyperlink URL needs to be updated and then it's good to go!

File: app/web/features/profile/edit/EditProfile.tsx (hunk 3 above)
Comment (aapeliv) [reply app/web/features/profile/edit/EditProfile.tsx]: Thanks for noticing! Fixed!

**Comment by aapeliv** [comment]
Merging this as it was previously approved code-wise and was just pending wording/help center updates!

**Pull request merged**